.travis.yml

```diff
@@ -38,15 +38,9 @@ jobs:
       script:
         - scripts/lint-text.sh --require-all
         - scripts/lint-go.sh
-        - scripts/test-go.sh
-
-    - name: rbdplugin
-      script:
-        - make rbdplugin
-
-    - name: cephfsplugin
-      script:
-        - make cephfsplugin
+        - scripts/test-go.sh || travis_terminate 1;
+        - make rbdplugin || travis_terminate 1;
+        - make cephfsplugin || travis_terminate 1;
```

Comment:
Let it finish building all artifacts and if there were any failures, it can then exit post catching the same. If there are failures in this phase "static-check-make" then when the next phase is added (say running the CI), then the early exit helps, but I still suggest that we finish all jobs and catch all failures from **this** phase, as that would help contributors.